docs/binary.md

```diff
@@ -639,6 +640,21 @@ When interacting with the XML format, care must be taken because `UniqueId` is s
 
 When an array of `UniqueId` values is present, the bytes are subject to [byte interleaving](#byte-interleaving).
 
+### Font
+**Type ID `0x20`**
+
+The `Font` type is a struct composed of two `String` values, a `u8`, and a `u16`:
+
+| Field Name   | Format              | Value                           |
+| :----------- | :------------------ | :------------------------------ |
+| Family       | [`String`](#string) | The font family content url     |
+| Weight       | `u16`               | The weight of the font          |
+| Style        | `u8`                | The style of the font           |
+| CachedFaceId | [`String`](#string) | The cached content url of the ttf file |
```

Comment:
```suggestion
| Field Name   | Format              | Value                                  |
|:-------------|:--------------------|:---------------------------------------|
| Family       | [`String`](#string) | The font family content URI            |
| Weight       | `u16`               | The weight of the font                 |
| Style        | `u8`                | The style of the font                  |
| CachedFaceId | [`String`](#string) | The cached content URI of the TTF file |
```
In the spec files, we prefer to have the delimiter row (the second one) not have leading or trailing spaces, contrary to the rest of the table. Also, the `Value` column should be aligned to the length of the max description, which in this case is the description of `CachedFaceId`.

Additionally, we capitalize acronyms and it's more accurate to refer to `Family` and `CachedFaceId` as URIs, since Roblox's `Content` type isn't necessarily a valid URL.